README.md

```diff
@@ -193,14 +193,14 @@ v8port/patches/. Adjust the path to v8port to match your environment. Example:
 This is a large patch which updates multiple portions of the codebase.
 ```
 $ cd $HOME/zopen/dev/v8base/v8
-$ git apply ../../v8port/patches/git.20231122.fce38915e4.diff
+$ git apply ../../patches/git.20231122.fce38915e4.diff
```

Comment:
Does this mean you have the v8port patches directly under `$HOME/zopen/dev/patches/`? Because [this line](https://github.com/ZOSOpenTools/v8port/pull/11/files#diff-5e6ba787ffa16b862d9020a87823ad745f43c9d7ecd0e030657d3e2c81d72b7dR66) indicates there is a `$HOME/zopen/dev/v8port/` that should contain the `patches` directory.

Reply:
Yes, the dir structure is like this:

```
v8port
  \_ v8base
       \_ v8
```